bookshop/sqlalchemy/model_to_dict.py

```diff
@@ -98,3 +95,20 @@ def _recurse_on_model_or_list(
             paths=next_paths,
             return_immediately=return_immediately,
         )
+
+
+def deep_merge(dict1, dict2):
```

Comment:
If this is copied from somewhere could you add a link to where it was copied from. It may have some useful context if someone it wanting to change it in the future.